### PR TITLE
Added linters

### PR DESCRIPTION
**Pull request by EvansSnave**
added hintrc, stylelint and linters

### CODE REVIEW

**Comment by EvansSnave** [comment]
Solved linter issues , customized readme and added test.md to gitignore

File: README.md [hunk 1]
```diff
@@ -1 +1,257 @@
...
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
```
Comment (Johnadibe) [inline README.md:66]: > - [x] Please kindly specify the tech stack used to build this project. It is a best practice that can help others understand, use, and contribute to your project.😊

File: README.md [hunk 2]
```diff
@@ -1 +1,257 @@
...
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Johnadibe) [inline README.md:85]: - [x] Please kindly remove the live demo section since you haven't deployed your project yet.👍

File: README.md [hunk 3]
```diff
@@ -1 +1,257 @@
...
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
```
Comment (Johnadibe) [inline README.md:229]: - [x] Please kindly add an acknowledgment to your project.✔ e.g _I would like to appreciate [Microverse](https://www.microverse.org/) for providing the resources and the platform for us to be a Professional full-stack developer._

File: README.md [hunk 4]
```diff
@@ -1 +1,257 @@
...
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```
Comment (Johnadibe) [inline README.md:254]: - [x] I noticed that your MIT license link is broken. Kindly fix that👍

File: README.md [hunk 5]
```diff
@@ -1 +1,257 @@
...
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (Johnadibe) [inline README.md:202]: - [x] Please kindly add  feature feature(s) you would like to add to the project.☺

**Review by Johnadibe — COMMENTED** [review]
# STATUS: REQUIRED CHANGES ♻️
### Hi @EvansSnave 👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Highlights
✅ You used GitHub Flow correctly 👍🏽
✅ Added .HTML and .css files 👍🏽


## Required Changes ♻️
> - [ ] Please kindly use one pull request until approval and not merge before approval next time.👍. You can open a new pull request from the feature branch to the main for re-review and apply the requested changes from this closed PR👍.

> - [ ] Please kindly delete all the files from the main branch and ensure that the feature branch has all the files that you want to merge into the main branch. This will help to keep the codebase clean and organized☺.
> 
> evans-main
> 

> - [ ] I noticed that the pull request you submitted didn't include a very detailed description. It's important to write a descriptive pull request that explains what changes you've made, why you made them, and how they benefit the project. This helps other team members review your code more effectively and understand the purpose of the changes you've made. Please include a detailed description in your pull requests👇.👍 

evans-description

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by EvansSnave** [comment]
Thank you very much for your review! I just corrected it and I think I fixed the issues

**Pull request merged**